### ISSUE TITLE
schema option defaultValue not working as expected

### ISSUE THREAD

**Issue by PolGuixe** [VulcanJS/Vulcan#1825]
I am trying to add a new field to users. For testing purposes I am trying to give it a default value, however the option doesn't seem to be working. 

```
import Users from 'meteor/vulcan:users';

Users.addField({
  fieldName: 'organitzations',
  fieldSchema: {
    type: String,
    optional: true,
    viewableBy:['members'],
    defaultValue: 'test',
  }
});
```
When I am fetching this field is returned null... 🤔

**Comment by MHerszak** [comment]
Hi, this might not the answer you expect but I am using `onInsert`, `onEdit` to add default values. Check out the categories schema in the forum example. Not sure if that helps.

**Comment by PolGuixe** [comment]
@MHerszak I am also using `onInsert` :).

I just wanted to report that `defaultValue` wasn't working for me.

**Comment by SachaG** [comment]
This goes back to the fact that when you define a schema, SimpleSchema takes out properties like `defaultValue`, etc. from the final field objects (there's a way to get them but it wasn't documented at the time), so I couldn't do `field.defaultValue` to auto-populate forms. I ended up adding a new `default` property that does the same thing, but not being part of SimpleSchema's own APIs it doesn't get any special treatment and is easier to deal with. 

TL;DR: we're using `default` instead of `defaultValue` in order to bypass SimpleSchema's own weird internal logic.
